photon-lib/src/main/native/cpp/photonlib/PhotonPoseEstimator.cpp

```diff
@@ -359,11 +359,16 @@ std::optional<EstimatedRobotPose> PhotonPoseEstimator::MultiTagPnpStrategy(
     std::optional<cv::Mat> distCoeffs) {
   using namespace frc;
 
+  // Need at least 2 targets
   if (!result.HasTargets() || result.GetTargets().size() < 2) {
     return Update(result, std::nullopt, std::nullopt,
                   this->multiTagFallbackStrategy);
   }
 
+  if (!camMat || !distCoeffs) {
+    return std::nullopt;
```

Comment:
Also needs to be changed to `return Update(result, std::nullopt, std::nullopt,
                  this->multiTagFallbackStrategy);`